src/Compilers/CSharp/Portable/Symbols/TypeSymbolExtensions.cs

```diff
@@ -1964,5 +1965,13 @@ internal static int TypeToIndex(this TypeSymbol type)
                 default: return -1;
             }
         }
+
+        internal static void CheckRecordLangVersion(this TypeSymbol type, DiagnosticBag diagnostics, TypeSyntax syntax, Location location)
+        {
+            if (type.IsErrorType() && syntax is IdentifierNameSyntax name && name.Identifier.Text == SyntaxFacts.GetText(SyntaxKind.RecordKeyword))
```

Comment:
>Text [](start = 93, length = 4)

Why use Text? I think we usually use Value and ValueText properties, or something like that.